### PR TITLE
[TM]Adding your first layer: make the steps more atomic

### PR DESCRIPTION
**Pull request by DelazJ**
Add more screenshots/details and fix distortion of first screenshot

Based on discussion in Jared Morgan [review of training manual](https://docs.google.com/document/d/1_0CalcMFFUtM0UxhLareUWNEZZVxslqynZn5fmLKkDQ/edit#heading=h.ijq8tpjxm5ma)

### Description

Goal:

Ticket: #

- [x] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

File: source/docs/training_manual/introduction/preparation.rst [hunk 1]
```diff
@@ -69,14 +87,14 @@ your work.
...
-data sets from the same folder (:file:`exercise_data/shapefile`) to the map.
+datasets from the same folder (:file:`exercise_data/shapefile`) to the map.
 
 :ref:`Check your results <interface-preparation-1>`
 
 |IC|
 -------------------------------------------------------------------------------
 
-You've learned how to add a layer and create a basic map!
+You've learned how to add a layer from a shapefile dataset and create a basic map!
```
Comment (havatv) [inline source/docs/training_manual/introduction/preparation.rst]:
```suggestion
You've learned how to add a layer from a Shapefile dataset and create a basic map!
```

File: source/docs/training_manual/introduction/preparation.rst [hunk 2]
```diff
@@ -29,32 +29,50 @@ Let's get started right away!
...
-#. Click the :guilabel:`Open Data Source Manager` button: |dataSourceManager|.
+
+   .. image:: img/add_blank_qgis.png
+      :align: center
+      :width: 100 %
+
+   |
+
+#. The :guilabel:`Data Source Manager` dialog allows you to choose the data to
+   load depending on the data type. We'll be using it to load our dataset:
```
Comment (ghtmtt) [inline source/docs/training_manual/introduction/preparation.rst]: We'll use it?

File: source/docs/training_manual/introduction/preparation.rst [hunk 3]
```diff
@@ -29,32 +29,50 @@ Let's get started right away!
...
-   Dataset(s)`.
-#. Navigate to the file :file:`exercise_data/shapefile/protected_areas.shp`
-   (in your course directory).
-#. Select the file and click :guilabel:`Open`. You will see the original dialog,
-   but with the file path filled in.
+   |
 
-   .. image:: img/add_vector_dialog.png
-      :align: center
+#. Load the :file:`protected_areas.shp` dataset vector file:
```
Comment (ghtmtt) [inline source/docs/training_manual/introduction/preparation.rst]: ~dataset~

File: source/docs/training_manual/introduction/preparation.rst [hunk 4]
```diff
@@ -29,32 +29,50 @@ Let's get started right away!
...
+   
+   #. Click on the :guilabel:`Vector` tab.
+   #. Enable the |radioButtonOn|:guilabel:`File` source type.
+   #. Press the :guilabel:`...` button next to :guilabel:`Vector Dataset(s)`.
+   #. Select the :file:`exercise_data/shapefile/protected_areas.shp` file
+      in your course directory.
```
Comment (ghtmtt) [inline source/docs/training_manual/introduction/preparation.rst]: course or training? not sure about that

File: source/docs/training_manual/introduction/preparation.rst [hunk 5]
```diff
@@ -69,14 +87,14 @@ your work.
 -------------------------------------------------------------------------------
 
 Repeat the steps above to add the :file:`places.shp` and :file:`rivers.shp`
-data sets from the same folder (:file:`exercise_data/shapefile`) to the map.
+datasets from the same folder (:file:`exercise_data/shapefile`) to the map.
```
Comment (ghtmtt) [inline source/docs/training_manual/introduction/preparation.rst]: ~datasets~. Maybe best to call them layers?

File: source/docs/training_manual/introduction/preparation.rst [hunk 6]
```diff
@@ -69,14 +87,14 @@ your work.
...
 
 :ref:`Check your results <interface-preparation-1>`
 
 |IC|
 -------------------------------------------------------------------------------
 
-You've learned how to add a layer and create a basic map!
+You've learned how to add a layer from a Shapefile dataset and create a basic map!
```
Comment (ghtmtt) [inline source/docs/training_manual/introduction/preparation.rst]: You've learned how to add a Shapefile layers and create a basic map!

File: source/docs/training_manual/introduction/preparation.rst (hunk 3 above)
Comment (havatv) [reply source/docs/training_manual/introduction/preparation.rst]: I suggest that we avoid using "file" for Shapefile format datasets, as these datasets consist of several files.  So in this case, I would just use "dataset" (and remove "vector file"), or "vector dataset".

File: source/docs/training_manual/introduction/preparation.rst (hunk 6 above)
Comment (havatv) [reply source/docs/training_manual/introduction/preparation.rst]: I prefer @DelazJ's proposal (or we can just leave it as it was: "You've learned how to add a layer and create a basic map!").

Comment (DelazJ) [reply source/docs/training_manual/introduction/preparation.rst]: > I prefer @DelazJ's proposal

Not mine, I just picked it from the google docs, like most of  the changes here

File: source/docs/training_manual/introduction/preparation.rst (hunk 3 above)
Comment (DelazJ) [reply source/docs/training_manual/introduction/preparation.rst]: `vector dataset` is good to me. We are at the beginning of the training manual so vector is not superfluous to ensure that trainees get what it's

**Pull request merged**